src/collective/vdexvocabulary/treevocabulary.py

```diff
@@ -106,12 +116,12 @@ def _createTermTree(self, element, terms, _):
                 # i18n message id for title/caption
                 _(
                     "caption|%s" % identifier,
-                    default=default_title or identifier.decode("utf-8"),
+                    default=default_title or identifier,
```

Comment:
This should be `safe_unicode(default_title or identifier)` it seems (before it was decoded).